examples/python/interceptors/headers/helloworld_pb2_grpc.py

```diff
@@ -0,0 +1,46 @@
+# Generated by the gRPC Python protocol compiler plugin. DO NOT EDIT!
+import grpc
+
+import helloworld_pb2 as helloworld__pb2
+
+
+class GreeterStub(object):
+  """The greeting service definition.
+  """
```

Comment:
For single-line doc strings, the closing triple-quote goes at the end of the single line.

(Unless `yapf` moves it back onto its own line?)